debian/prebuild.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+curl -LsSf https://www.tarantool.io/release/1.10/installer.sh | sudo bash
```

Comment:
Nit: The change is titled 'deb: disable running tests on build' and it is a bit unexpected to see this addition here. Please, rewrite the commit title or split the changes.

Reply:
Moved to the separate commit